noxfile.py

```diff
@@ -3,40 +3,43 @@
 nox.options.stop_on_first_error = True
 nox.options.reuse_existing_virtualenvs = False
 
-nox.options.sessions = ['test', 'lint', 'docs', 'install']
+nox.options.sessions = ['lint', 'test', 'coverage', 'docs']
 
 source_files = ("planet", "examples", "tests", "setup.py", "noxfile.py")
 
 
-@nox.session(python=["3.7", "3.8", "3.9"])
-def test(session):
-    # must include -e to allow coverage file update
+@nox.session
+def coverage(session):
     session.install("-e", ".[test]")
+    session.run('coverage',
+                'run',
+                '-m',
+                'pytest',
+                '-qq',
+                '--no-header',
+                '--no-summary',
+                '--no-cov',
+                '--ignore',
+                'examples/')
+    session.run('coverage', 'report')
 
-    options = session.posargs
-    if '-k' in options:
-        options.append('--no-cov')
-    session.run('pytest', '--ignore', 'examples/', '-v', *options)
```

Comment:
The options are needed here to allow just running a test on one file or one file's test, aka `nox -s test-3.7 -- tests/unit/test_specs.py`. Please reinstate them.